db/schema.rb

```diff
@@ -11,7 +11,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150823122126) do
+ActiveRecord::Schema.define(version: 20151011163832) do
+
+  create_table "books", force: :cascade do |t|
+    t.string   "title"
+    t.string   "author"
+    t.string   "synopsis"
+    t.string   "image_url"
+    t.boolean  "available"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```

Comment:
More model stuff but there can be more book states. Can be modeled with AR enum. https://github.com/DUCSS/ducss-site-old/blob/master/project/library/models.py shows some of the model choices made for the old version